integration-tests/build.gradle

```diff
@@ -5,6 +5,7 @@ subprojects {
     apply plugin: 'java'
 
     ext.enclaveMode = findProperty("enclaveMode")?.toString()?.toLowerCase() ?: "simulation"
+    ext.runtimeType = findProperty("runtimeType")?.toString() ?: "GraalVM"
```

Comment:
Can we guarantee that `graalvm` is the same as `GraalVm` or `GraalVM`, etc?

Reply:
At the moment it's strict, you have to use the exact capitalisation, but the error message contains the correct capitalisation if you get it wrong:

```
> 'gramine' is not a valid enclave runtime type.
  Valid runtime types are: [Gramine, GraalVM].
```

Can uppercase/lowercase this if you feel it's necessary.

Reply:
I'd personally keep consistent with the `enclaveMode` now and make it lowercase. If not, I expect time wasted in remembering if it was `GraalVM` or `GraalVm`

Reply:
Let's make it case insensitive. It's easier for the user.

Reply:
Gotcha, will make it case insensitive